Libraries/NavigationExperimental/NavigationAnimatedView.js

```diff
@@ -212,6 +212,7 @@ class NavigationAnimatedView extends React.Component {
     const {renderOverlay} = this.props;
     if (renderOverlay) {
       return renderOverlay(
+        this.props.navigationState,
```

Comment:
Did you test every call site of renderOverlay? It also doesn't look like flow will pass after this change.